src/auth-service/bin/jobs/message-consumer.js

```
@@ -0,0 +1,552 @@
+      { groupId: constants.UNIQUE_CONSUMER_GROUP }
+    );
+
+    logger.info(
+      `Message consumer subscribed successfully using ${messagingService.getActiveBrokerType()} broker`
+    );
+
+    // Log subscription results for each broker
+    if (subscriptionResult && subscriptionResult.results) {
+      subscriptionResult.results.forEach((result) => {
+        if (result.success) {
+          logger.info(
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)